src/fare-contracts/FareContractInfo.tsx

```diff
@@ -147,7 +146,7 @@ export const FareContractInfoDetails = (
   );
 
   const isValidOrSentFareContract: boolean =
-    isValidFareContract(status) || isSentFareContract(status);
+    isValidFareContract(status) || status === 'sent';
```

Comment:
we may use a `const isStatusSent = status === 'sent'` and reuse it.

Reply:
There's so much checking of sent fare contract that I think it's better to check it manually instead of making more functions 😆 , though this one I think I can do it.